boltdesignsystem/bolt
# DS-298 docs.pega.com design

## Jira

https://pegadigitalit.atlassian.net/browse/DS-298

## Summary

Updates multiple components and layouts to enhance docs.pega.com site design.

## Details

### Holy Grail Layout

- Finalized the layout to be released
- Rewrote docs to reflect changes

### Site Layout

- Fixed a few doc typos

### Page Header

- Re-organized docs
- Fixed schemas and added validations
- Added `flat` prop to nav li template
- Removed obsolete props from nav ul template
- Added `align_site_nav_items` prop to primary nav template
- Removed `theme` prop from main template as theming has grown, use appropriate `t-bolt-*` classes instead
- Added `full_width` prop to main template

### Side Nav

- Finalized the component to be released

### Pagination

- Fixed a CSS bug where the current item is missing padding

### Typeahead

- Updated default placeholder text to be more precise plain English

### Listing Teaser

- Added a CSS to reset font-weight so it doesn't inherit from parent container

### Community Pages

- Added mockups for Doc layout and Search layout

### Mode CSS Vars

- Fixed a few theme colors for better contrast
- Fixed a bug where the theme background color doesn't render correctly

## How to test

Run the branch locally and check the docs for all the components and layouts listed in details. Make sure all work according to the schemas.
